### ISSUE TITLE
FileAPI pollyfil _files[0].stream has no value

### ISSUE THREAD

**Issue by istian** [balderdashy/skipper#39]
Hi,

I was working on a file upload page which I need to IE8 browser which lead me to using FileApi pollyfill. When uploading file, the controller fetches data via req.file('file_element')._files[0].stream but using the FileApi the "stream" property is missing or undefined, causing error "ETIMEOUT: An Upstream (`undefined`) timed out waiting for file(s). No files were sent after waiting 500ms".

Could someone shed some light? I'm not sure where is the issue?

Thanks in advance.

**Comment by sailsbot** [comment]
Thanks for posting, @istian.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
